pkg/settings/setting.go

```diff
@@ -47,6 +47,7 @@ var (
 	InstallUUID                       = NewSetting("install-uuid", "")
 	InternalServerURL                 = NewSetting("internal-server-url", "")
 	InternalCACerts                   = NewSetting("internal-cacerts", "")
+	IsRKE                             = NewSetting("is-rke", "")
```

Comment:
Where is this getting set? Should it be added to the map of settings to pass to downstream?

Reply:
I think I forgot to check in that part, let me look

Reply:
This is already set here https://github.com/rancher/rancher/blob/release/v2.6/pkg/systemtemplate/template.go#L201